packages/google_maps_flutter/google_maps_flutter_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.0+4
+
+* Fix crash when InfoWindow title/snippet has links. [Issue](https://github.com/flutter/flutter/issues/67854).
```

Comment:
Probably should mention the dep updates, which is the source of the fix.

Reply:
Updated to something more descriptive!